cmd/soci-snapshotter-grpc/main.go

```diff
@@ -118,16 +120,23 @@ func main() {
 	// Snapshotter should use "github.com/containerd/log" otherwise
 	// logs are always printed as "debug" mode.
 	golog.SetOutput(log.G(ctx).WriterLevel(logrus.DebugLevel))
-	log.G(ctx).WithFields(logrus.Fields{
-		"version":  version.Version,
-		"revision": version.Revision,
-	}).Info("starting soci-snapshotter-grpc")
+	if !*dumpConfig {
+		log.G(ctx).WithFields(logrus.Fields{
+			"version":  version.Version,
+			"revision": version.Revision,
+		}).Info("starting soci-snapshotter-grpc")
```

Comment:
I think it might just be fine to move this log below the configuration parsing and dump exit point. WDYT?